zaas-client/src/main/java/org/zowe/apiml/zaasclient/service/ZaasClient.java

```diff
@@ -106,6 +107,21 @@ public interface ZaasClient {
      */
     ZaasToken query(HttpServletRequest request) throws ZaasClientException;
 
+    /***
+     * Return details about the provided OIDC token by validating it against an API Mediation Layer instance.
+     *
+     * This method supports simple boolean validation against Zowe v2 and v2 as of v3.0
+     *
+     * A successful validation means the token was created using the same settings as found in the target API Mediation Layer.
```

Comment:
Perhaps this is non-standard for Java API doc, but should it be explicitly stated that "successful validation" means that `ZaasOidcValidationResults.isValid()` is true?

Reply:
It can be added to the javadoc description, no problem with that